### DISCUSSION TITLE
Version 0.14.0 of glowroot-agent not available on mvnrepository.com

### DISCUSSION THREAD

**Discussion by lme-atolcd** [glowroot/glowroot#1005] [General]
Hello,

Latest stable release 0.14.0 of glowroot-agent is not available on mvnrepository.com. Previous versions (up to 0.14.0-beta.3) are available: https://mvnrepository.com/artifact/org.glowroot/glowroot-agent

Could it be possible to upload 0.14.0 to mvnrepository.com?
Thanks.

**Comment by Neyos1997** [comment]
Hello @lme-atolcd  
do you have any documentation how to put glowroot-agent to work  including it as a maven dependency ?  
i don't know which resources should include in my project as well with the dependency ofc . 
Else i  cloned the repo and didn't manage to do some changes to the ui   , idk the deployable jar is under /dist , the ui is seperate how to link these two jars , i literally have no idea . 
I am desperate to the point to ask for other people 's support in comments section . 

Thanks a lot

**Reply by lme-atolcd** [reply]
Hi @Neyos1997

We do not use glowroot-agent as a maven dependency in a maven project.

In our workflown, we have a Dockerfile that download glowroot-agent from mvnrepository.com and add it to our Java application. We use mvnrepository.com to dowload glowroot-agent because we have an internal Maven proxy used to cache dowloaded jars from mvnrepository.com.
As glowroot-agent 0.14.0 is not yet available on mvnrepository.com, our workflow is broken. As a temporary fix, our Dockerfile download glowroot-agent straight from the github releases.

Feel free to open a new discussion for your questions about modifying the UI or using glowroot-agent as a maven dependency.

**Reply by Neyos1997** [reply]
that was highly appreciated thanks a lot
